config/config.default.yaml

```diff
@@ -1215,6 +1215,8 @@ plotting:
     residential urban decentral water tanks discharger: '#baac9e'
     services rural water tanks discharger: '#bbc2b8'
     services urban decentral water tanks discharger: '#bdd8d3'
+    water pits: '#cc826a'
```

Comment:
pit storage should be only added within district heating systems, i.e. as urban central hot water pit